src/workspace-data/data-table/wds/WDSContent.test.ts

```diff
@@ -213,4 +216,26 @@ describe('WDSContent', () => {
       expect(deleteColumnMock).toHaveBeenCalledWith(expect.any(AbortSignal), 'marble', 'color');
     });
   });
+
+  describe('edit column', () => {
+    it('edit field icon is present for types that support editing', async () => {
+      const { props } = setup({
```

Comment:
Nit: preferably, all tests have arrange/act/assert sections commented.

```suggestion
      // Arrange
      const { props } = setup({
```